lib/solidus_graphql_api.rb

```diff
@@ -9,3 +9,4 @@
 require 'solidus_graphql_api/version'
 require 'solidus_graphql_api/engine'
 require 'solidus_graphql_api/configuration'
+require 'solidus_graphql_api/railtie' if defined?(Rails)
```

Comment:
I'm not sure if I can remove the `if defined?(Rails)` check. WDYT?

Reply:
Yes, you can avoid that, since Rails has been already loaded at that point.